pep257.py

```diff
@@ -254,6 +268,47 @@ def parse_docstring(self):
             return docstring
         return None
 
+    def parse_decorators(self):
+        """Called after first @ is found.
+
+        Build list of current decorators and return last parsed token,
```

Comment:
This doesn't really return the last parsed token - the docstring should be updated.